src/main/java/org/openrewrite/java/spring/ChangeSpringPropertyKey.java

```diff
@@ -96,4 +107,62 @@ private String exceptRegex() {
                 "" :
                 "(?!(" + String.join("|", except) + "))";
     }
+
+    private J.Annotation mapArgs(J.Annotation a) {
+        if (a.getArguments() != null) {
+            a = a.withArguments(ListUtils.map(a.getArguments(), arg -> {
+                if (arg instanceof J.Literal) {
+                    J.Literal literal = (J.Literal) arg;
+                    if (literal.getValue() instanceof String) {
+                        String value = (String) literal.getValue();
+                        if (value.contains(oldPropertyKey)) {
+                            Pattern pattern = Pattern.compile("\\$\\{(" + quote(oldPropertyKey) + "(?:\\.[^.}:]+)*)(((?:\\\\.|[^}])*)\\})");
```

Comment:
I'd sort of expected to see reuse of the `exceptRegex` here as well; was that a conscious change here?

Reply:
Good call!